website/docs/shadermodel.md

```diff
@@ -1,29 +1,29 @@
-shadermodel - This page was auto-generated. Feel free to help us improve the documentation by creating a pull request.
+Specifies the shader model.
 
 ```lua
-shadermodel (value)
+shadermodel "value"
```

Comment:
Let's use `shadermodel ("value")` to keep consistent with other docs - (e.g.) https://premake.github.io/docs/objdir/

Reply:
I'll make the relevant changes as requested.  Hey no problem, I've never been good at documentation but just wanted to make these small fixes to make things a little more clear for the future.

Reply:
Appreciate all help with docs. When you get those changes out, I'll rereview and merge them so we can get it out.

Reply:
Just poking this for line 39 as well.